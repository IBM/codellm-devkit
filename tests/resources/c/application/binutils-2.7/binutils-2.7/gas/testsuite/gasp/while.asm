	donkey
bar	.ASSIGNA	0
	.AWHILE	\&bar LT 5	
	HI BAR IS \&bar
foo	.ASSIGNA	0
	.AWHILE	\&foo LT 2
	HI BEFORE
	.AREPEAT	2
	HI MEDIUM	\&foo \&bar
	.AENDR
	HI AFTER
foo	.ASSIGNA	\&foo + 1	
	.AENDW
bar	.ASSIGNA	\&bar + 1	
	AND ITS NOW \&bar
	.AENDW
	.END

