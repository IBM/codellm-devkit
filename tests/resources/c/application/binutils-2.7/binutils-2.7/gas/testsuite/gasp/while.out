!	donkey
	donkey
!bar	.ASSIGNA	0
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	HI BAR IS \&bar
	HI BAR IS 0
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 0
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 0
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 0
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 0
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
	AND ITS NOW 1
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	HI BAR IS \&bar
	HI BAR IS 1
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 1
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 1
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 1
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 1
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
	AND ITS NOW 2
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	HI BAR IS \&bar
	HI BAR IS 2
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 2
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 2
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 2
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 2
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
	AND ITS NOW 3
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	HI BAR IS \&bar
	HI BAR IS 3
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 3
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 3
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 3
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 3
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
	AND ITS NOW 4
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	HI BAR IS \&bar
	HI BAR IS 4
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 4
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	0 4
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!	HI BEFORE
	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 4
!	.AREPEAT	1
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI MEDIUM	\&foo \&bar
	HI MEDIUM	1 4
!	HI AFTER
	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
	AND ITS NOW 5
!	.AWHILE	\&bar LT 5	
!	HI BAR IS \&bar
!foo	.ASSIGNA	0
!	.AWHILE	\&foo LT 2
!	HI BEFORE
!	.AREPEAT	2
!	HI MEDIUM	\&foo \&bar
!	.AENDR
!	HI AFTER
!foo	.ASSIGNA	\&foo + 1	
!	.AENDW
!bar	.ASSIGNA	\&bar + 1	
!	AND ITS NOW \&bar
!	.AENDW
!	.END
